src/js/controllers/LoginController.js

```diff
@@ -0,0 +1,12 @@
+function LogController($scope, $http) {
+    $scope.login = () => {
+        $http.post('http://api.kitchen.support:8000/login',
```

Comment:
This will be 1) on port 80, so, no port, and 2) `/accounts/login` rather than `/login`